code/game/antagonist/outsider/ert.dm

```diff
@@ -1,122 +0,0 @@
-	..()
-	welcome_text = "As member of the Emergency Response Team, you answer only to your leader and [GLOB.using_map.company_name] officials."
-
-/datum/antagonist/ert/greet(var/datum/mind/player)
-	if(!..())
-		return
-	to_chat(player.current, "You should first gear up and discuss a plan with your team. More members may be joining, don't move out before you're ready.")
-
-/datum/antagonist/ert/proc/add_candidate(var/mob/source)
-	candidates += source
-
-/datum/antagonist/ert/proc/pick_candidates()
-
-	if(!outfits || outfits.len > candidates.len)
-		message_admins("Ert aborted. Not enough candidates.", 1)
-		log_admin("Ert aborted. Not enough candidates.")
-		return 0
-
-	for(var/i = 0; i <= outfits.len; i++)
-		if(candidates.len == 0)
-			break
-		var/candidate = pick(candidates)
-		create_default(candidate)
-		candidates -= candidate
-	candidates = list()
-
-
-/datum/antagonist/ert/equip(var/mob/living/carbon/human/player)
-	if(!..())
-		return 0
-	if(!outfits.len)
-		return 0
-	var/list/outfits_temp = outfits
-	var/decl/hierarchy/outfit/ert_outfit = outfit_by_type(outfits_temp[1])
-	if(!ert_outfit)
-		return 0
-	dressup_human(player, ert_outfit)
-	outfits_temp -= outfits_temp[1]
-	return 1
-
-/datum/antagonist/ert/kellion
-	id = "Kellionteam"
-	role_text = "Maintenance Response Team"
-	role_text_plural = "Maintenance Response Team"
-	leader_welcome_text = "As leader of the Emergency Response Team, you are part of the Kellion Response Team, and are there with the intention of restoring normal operation to the vessel or the safe evacuation of crew and passengers. You should, to this effect, aid the Commanding Officer or ranking officer aboard in their endeavours to achieve this."
-	landmark_id = "kellionteam"
-	hard_cap = 5
-	hard_cap_round = 5
-	initial_spawn_req = 5
-	initial_spawn_target = 5
-	outfits = list(
-		/decl/hierarchy/outfit/isaac,
-		/decl/hierarchy/outfit/kellion_sec,
-		/decl/hierarchy/outfit/kellion_sec,
-		/decl/hierarchy/outfit/kellion_sec_leader,
-		/decl/hierarchy/outfit/kendra)
-
-/datum/antagonist/ert/unitologists
-	id = "Unitologiststeam"
-	role_text = "Unitologist"
-	role_text_plural = "Unitologists"
-	hard_cap = 5
-	hard_cap_round = 5
-	initial_spawn_req = 5
-	initial_spawn_target = 5
-	antag_text = "You are part of a new religion which worships strange alien artifacts, believing that only through them can humanity truly transcend. You have been blessed with a psychic connection created by the <b>marker</b>, one of these artifacts. Serve the marker's will at all costs by bringing it human sacrifices and remember that its objectives come before your own..."
-	leader_welcome_text = "You are the leader of this response team. Work with the marker instead of against it."
-	landmark_id = "unitologiststeam"
-	antaghud_indicator = "hudunitologist" // Used by the ghost antagHUD.
-	antag_indicator = "hudunitologist"// icon_state for icons/mob/mob.dm visual indicator.
-	outfits = list(
-		/decl/hierarchy/outfit/faithful,
```

Comment:
i hope all this stuff has been transfererred over

Reply:
Yep.